### DISCUSSION TITLE
How do I install EarTrumpet?

### DISCUSSION THREAD

**Discussion by Leviathan556** [File-New-Project/EarTrumpet#798] [Q&A]
So I've downloaded EarTrumpet from here cuz for some reason the Microsoft Store doesn't work anymore and cannot install the app. There is no .exe file with which I can install the app. If anyone on here knows any solutions to this I'd be very thankful. Cheers

**Comment by riverar** [comment]
@Leviathan556 Microsoft Store is our main distribution method (do let us know what error you're seeing). But there's also Chocolatey and winget if desired.

**Reply by Leviathan556** [reply]
For some reason I stopped receiving updates for my windows which pretty much disabled mc store. And I really think this app would make my life easier. If you can. Can you give me the two links to EarTrumpet?
